File: foxogram-common/src/main/java/su/foxogram/dto/gateway/response/HeartbeatACKDTO.java
package su.foxogram.dto.gateway.response;

import lombok.Getter;
import lombok.Setter;
import su.foxogram.constant.GatewayConstant;

@Getter
@Setter
public class HeartbeatACKDTO {

	private int op;

	public HeartbeatACKDTO() {
		this.op = GatewayConstant.Opcode.HEARTBEAT_ACK.ordinal();
	}
}

File: src/main/java/su/foxogram/dto/gateway/response/HeartbeatACKDTO.kt
package su.foxogram.dto.gateway.response

import lombok.Getter
import lombok.Setter
import su.foxogram.constant.GatewayConstant

@Getter
@Setter
data class HeartbeatACKDTO(
    val op: Int = GatewayConstant.Opcode.HEARTBEAT_ACK.ordinal
)
